js/rxjs.md

```diff
@@ -0,0 +1,24 @@
+# Bloomfire RxJS Style Guide
+
+*Very much a work in progress*
+
+## Table of Contents
+
+  1. [Observable Naming](#observables)
+
+## observables 
+
+  - Files containing observables should be named in camelCase.
+  - Observable classes should be capitalized and MixedCase.
+  - Observable classes should substitute a trailing $ for the word "observable."
+  - Observable instances should be camelCase.
+
+  ```jsx
+  import SomeAPICall$ from 'app/observables/someAPICallObservable';
+  ````
+
+  ```jsx
+  const someAPICall = app.getInstance(SomeAPICall$);
```

Comment:
Ok, I'm liking the following then:

> file: someObservable
> class: Some$
> instance: some$

which seems like a lot to remember, but it's consistent and distinguishes all 3 cases. Also, thanks for the input @nixonchris!